src/basic/ioprio.h

```diff
@@ -10,12 +10,13 @@
 /*
  * Gives us 8 prio classes with 13-bits of data for each class
  */
-#define IOPRIO_BITS             (16)
-#define IOPRIO_CLASS_SHIFT      (13)
-#define IOPRIO_PRIO_MASK        ((1UL << IOPRIO_CLASS_SHIFT) - 1)
+#define IOPRIO_BITS             16
+#define IOPRIO_N_CLASSES        8
+#define IOPRIO_CLASS_SHIFT      13
+#define IOPRIO_PRIO_DATA_MASK   ((1UL << IOPRIO_CLASS_SHIFT) - 1)
```

Comment:
hmm, do we really want to deviate from linux/ioprio.h for the naming of this?

Reply:
I seem to have overlooked the inconsistency with `linux/ioprio.h`. In retrospect, I think it would have been better not to change the names.